client/src/App.js

```diff
@@ -19,10 +19,15 @@ export default class App extends Component {
   };
 
   render() {
+
+    console.log('STATE: ', this.state.savedList)
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <div>
+          <Route exact path="/" component={MovieList} />
+          <Route path="/movies/:id" render={ (props) => <Movie {...props} addToSavedList={this.addToSavedList} />} />
```

Comment:
I know this is a little confusing, don't worry if it doesn't fully make sense at this point. Feel free to let me know if you have questions, but also know that this will likely be less useful/relevant when we look at implementing Redux :+1: